### ISSUE TITLE
New term request: Migratory arthritis

### ISSUE THREAD

**Issue by yankeetomer** [obophenotype/human-phenotype-ontology#5818]
**Preferred term label:**
Migratory arthritis

**Synonyms**
Migratory polyarthritis

**Definition (free text, please give PubMed ID)**
A form of arthritis characterized by gradual involvement of several joints, with involvement of some joints occurring following partial or full resolution of arthritis in joints with an earlier onset.

PMID: 31657698

> Migratory
At the onset of arthritis, initially only one or more joints are involved, which they improve completely after several days. Following this, another joint region is involved, and in this way, polyarthritis occurs gradually. Acute rheumatic fever (ARF) is a typical example of this pattern of arthritis. 

**Parent term (use hpo.jax.org/app)**
Arthritis 
HP:0001369
https://hpo.jax.org/app/browse/term/HP:0001369

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acute rheumatic fever
https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=EN&Expert=3099

PMID: 18810554
> Migratory polyarthritis is a common symptom encountered by primary care physicians. The differential diagnosis is broad and includes infectious causes, crystal induced arthropathy, rheumatoid arthritis, vasculitic syndromes, connective tissue disorders, and spondyloarthridies6.

**Your nano-attribution (ORCID)**
Tomer Talmy M.D.
Organization: Emedgene Technologies
ORCID: https://orcid.org/0000-0002-7440-8864

**Comment by pnrobinson** [comment]
added as HP:0033037
